test/runtests.jl

```diff
@@ -144,6 +184,51 @@ end
 const prob_pds_linmod_nonconservative_inplace = PDSProblem(linmodP!, linmodD!, [0.9, 0.1],
                                                            (0.0, 2.0), [5.0, 1.0];
                                                            analytic = f_analytic)
+
+# Function definitions for testset "Linear advection"
+# Functions are defined outside the testset to avoid failing allocation tests,
+# see https://github.com/SKopecz/PositiveIntegrators.jl/pull/89.
+# number of nodes
+N = 1000
+u0 = sin.(π * LinRange(0.0, 1.0, N + 1))[2:end]
+tspan = (0.0, 1.0)
+# in-place syntax for f
+function fdupwind!(du, u, p, t)
```

Comment:
Since these functions are now used for linear advection tests, maybe we can make their names a bit more concrete, e.g., `linear_advection_fd_upwind!` etc.?